config/config.go.template

```diff
@@ -4,4 +4,5 @@ const(
 	Username = "root"
 	Password = ""
 	Port = "3306"
+	AESKey = "f5ec76dvcd899dfe"

```

Comment:
Just put a `aaaa` 16 times or so.

Reply:
Key can be anything, but I can change it for the default.

Reply:
I know, but it seems like an actual key.